examples/fabric-bridge-app/fabric-bridge-common/RootEndpointOnlyAccessInterface.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "RootEndpointOnlyAccessInterface.h"
+
+using namespace chip;
+using namespace chip::app;
+
+namespace bridge {
+
+CHIP_ERROR RootEndpointOnlyAccessInterface::Init()
+{
+    AttributeAccessInterface * oldInterface = AttributeAccessInterfaceRegistry::Instance().Get(kRootEndpointId, mClusterId);
+    VerifyOrDie(oldInterface != this);
+    VerifyOrReturnError(oldInterface != nullptr, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mOriginalAttributeInterface == nullptr, CHIP_ERROR_INCORRECT_STATE);
+
+    mOriginalAttributeInterface = oldInterface;
+    AttributeAccessInterfaceRegistry::Instance().Unregister(mOriginalAttributeInterface);
```

Comment:
Can we just fix the core SDK impl to only register itself on the root endpoint, so we don't have to mess with it here?

Reply:
Yes, I will do that in #39470 which is what got me into fixing this in the first place.